kubectl-pg/cmd/root.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"os"
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "kubectl-pg",
+	Short: "A kubectl pg plugin",
+	Long:  `kubectl pg plugin for interaction with Zalando postgres operator.`,
```

Comment:
1. Change to "kubectl plugin for the Zalando Postgres operator."
2. Please add a docs' URL, see  `kubectl` help for example
> Find more information at: https://kubernetes.io/docs/reference/kubectl/overview/

Reply:
But which URL should i point to ?

Reply:
for now it suffices to point to this PR; in two weeks we will know where the plugin will live and update the URL